### PR TITLE
feat: rest api port parameter

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #1099 
Added a separate cmd parameter to specify port to start Artipie rest api on.

But in the case SSL is used, it seems we have to use port 443 and ignore this option...

### CODE REVIEW

File: src/main/java/com/artipie/VertxMain.java [hunk 1]
```diff
@@ -143,8 +148,10 @@ public static void main(final String... args) throws Exception {
         final Options options = new Options();
         final String popt = "p";
         final String fopt = "f";
-        options.addOption(popt, "port", true, "The port to start artipie on");
-        options.addOption(fopt, "config-file", true, "The path to artipie configuration file");
+        final String apiport = "apiport";
```
Comment (dgarus) [inline src/main/java/com/artipie/VertxMain.java:151]: "apiport" -> "api-port"

**Review by dgarus — CHANGES_REQUESTED** [review]

File: src/main/java/com/artipie/VertxMain.java (hunk 1 above)
Comment (olenagerasimova) [reply src/main/java/com/artipie/VertxMain.java:151]: @dgarus cmd option name cannot have dashes

Comment (dgarus) [reply src/main/java/com/artipie/VertxMain.java:151]: @olenagerasimova ok

**Review by dgarus — APPROVED** [review]
thanks!

**Review by andpopov — APPROVED** [review]
@olenagerasimova Looks good! Thanks

**Pull request merged**